tests/test_k8s.py

```diff
@@ -2,7 +2,19 @@
 from unittest.mock import Mock, patch, call, PropertyMock, create_autospec
 from kubernetes.client.models import V1Pod
 from calrissian.k8s import load_config_get_namespace, KubernetesClient, CalrissianJobException, PodMonitor, delete_pods
-from calrissian.k8s import CompletionResult
+from calrissian.k8s import CompletionResult, read_file
+
+
+class ReadFileTestCase(TestCase):
+
+    @patch('builtins.open')
+    def test_read_file(self, mock_open):
+        mock_result = Mock()
+        mock_open.return_value.read.return_value = mock_result
+        result = read_file('filename.txt')
```

Comment:
Not related to changes here but I was wondering if the file opened in read_file needs to be closed. https://github.com/Duke-GCB/calrissian/blob/169f70eb94adf470396d8d143c22ac92fcf967fa/calrissian/k8s.py#L21-L22

Reply:
You're right, it probably should. I'll wrap this in a `with` context.